velox/docs/functions/spark/misc.rst

```diff
@@ -15,6 +15,17 @@ Miscellaneous Functions
         SELECT at_least_n_non_nulls(2, 0, 1.0, NULL);  -- true
         SELECT at_least_n_non_nulls(2, 0, array(NULL), NULL);  -- true
 
+.. spark:function:: get_struct_field(struct, ordinal) -> T
+
+    Returns the value of nested subfield at position ``ordinal`` in the input ``struct``.
+    The input must be of row type and nested complex type is allowed.
+    The subfield position is specified by ``ordinal``.
+    If ``ordinal`` is negative or greater than the children size of ``struct``,
```

Comment:
> greater than the children size of ``struct``

equal to the children size is also invalid, right?

Reply:
not accurate. updated